R/count_teleconnections.R

```diff
@@ -155,7 +155,7 @@ count_watershed_teleconnections <- function(data_dir,
         if (city %in% sliced_cities) {
           get_raster_val_classes_byslice(watersheds_city, cropcover_USA, poly_slices, n_cores) -> cropcover_ids
```

Comment:
Why is this left in?  The conditional statement should now be inconsequential since we can process them very efficiently.  Also, if this is not needed, then we can reduce some of the arguments in:
```r
count_watershed_teleconnections <- function(data_dir,
                                            watersheds_file_path = "water/CWM_v2_2/World_Watershed8.shp",
                                            powerplants_file_path = "water/UCS-EW3-Energy-Water-Database.xlsx",
                                            crop_file_path = "land/2016_30m_cdls/2016_30m_cdls.img",
                                            dams_file_path = "water/nabd_fish_barriers_2012/nabd_fish_barriers_2012.shp",
                                            irrigation_file_path = "land/usa_demeter.csv",
                                            nlud_file_path = "land/usa_nlud_raster.tif",
                                            cities = NULL,
                                            poly_slices = 40,
                                            n_cores = 2){```